testing/libfixposix/APKBUILD

```diff
@@ -0,0 +1,37 @@
+# Maintainer: Will Sinatra <wpsinatra@gmail.com>
+# Contributor: Will Sinatra <wpsinatra@gmail.com>
+pkgname=libfixposix
+pkgver=0.4.3
+pkgrel=1.0
+pkgdesc="Thin wrapper over POSIX syscalls"
+url="https://github.com/sionescu/libfixposix"
+arch="x86_64"
```

Comment:
is it really only x86_64 ?

Reply:
Good point, I believe it's x86 as well, I should have tested that before making the PR.